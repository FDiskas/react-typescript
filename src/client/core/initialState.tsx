import { IItem } from '../components/checkbox/CheckBoxComponent';

export const initialState = {
    checkBoxItems: [
        { id: '1', label: 'Label 1', value: 'Value 1' },
        { id: '2', label: 'Label 2', value: 'Value 2' },
    ] as IItem[],
};
