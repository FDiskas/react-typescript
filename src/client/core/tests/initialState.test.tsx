import { initialState } from '../initialState';

describe('initialState.js', () => {
    it('matches snapshot', () => {
        expect(initialState).toMatchSnapshot();
    });
});
