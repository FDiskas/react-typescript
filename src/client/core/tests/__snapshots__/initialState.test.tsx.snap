// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`initialState.js matches snapshot 1`] = `
Object {
  "checkBoxItems": Array [
    Object {
      "id": "1",
      "label": "1",
      "value": "1",
    },
    Object {
      "id": "2",
      "label": "2",
      "value": "2",
    },
  ],
}
`;
